taxi/views.py

```diff
@@ -1,3 +1,15 @@
+from django.http import HttpResponse
 from django.shortcuts import render
+from taxi.models import Driver,Manufacturer, Car
 
-# Create your views here.
+
+def index(request:HttpResponse) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+    context = {
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+        "num_cars": num_cars
+    }
+    return HttpResponse(render(request, "taxi/index.html", context=context))
```

Comment:
The 'render' function already returns an HttpResponse object, so there is no need to wrap it in another HttpResponse. You should directly return the result of 'render'.